### PR TITLE
LPS-118474 Changed markup structure for breadcrumb to fix styles

### PR DESCRIPTION
**Pull request by dgomezg**

### CODE REVIEW

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Breadcrumb.es.js [hunk 1]
```diff
@@ -82,15 +82,18 @@ export default ({section}) => {
 	}, [buildBreadcrumbNodesData, section]);
 
 	return (
+
```
Comment (nhpatt) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Breadcrumb.es.js:85]: remove

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Breadcrumb.es.js [hunk 2]
```diff
@@ -82,15 +82,18 @@ export default ({section}) => {
...
 		<section className="align-items-center d-flex">
 			<div className="questions-breadcrumb">
-				<ol className="breadcrumb mb-0 ml-2">
-					{breadcrumbNodes.length > MAX_SECTIONS_IN_BREADCRUMB ? (
-						<ShortenedBreadcrumb />
-					) : (
-						<AllBreadcrumb />
-					)}
-				</ol>
+				<div className="questions-container">
```
Comment (nhpatt) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Breadcrumb.es.js:88]: this is very strange... a container that only contains the breadcrumb does not look logical. Can you use question-root to apply community styles?

Comment (dgomezg) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Breadcrumb.es.js:88]: The tricky part is that the while the breadcrumb background should be aligned to `questions-root` (100% width) the `ol` tag (The contents of the breadcrumb) should be aligned to `questions-container` width. I can modify the styles of our custom css in liferay-community-theme to apply the same styles of `questions-container` to `breadcrumb` but that way, imho, the spec of "background should be aligned to root, and contents aligned to container" from the design wouldn't be enforced by the app.

Comment (dgomezg) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Breadcrumb.es.js]: I've removed the questions-container and changed some of the bootstrap flexbox styles to fix the alignment later in custom portal theme

Comment (nhpatt) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Breadcrumb.es.js]: But if you want to have `questions-container` values, you should wrap everything (and not only the breadcrumb and then close it) in it, right?

**Comment by nhpatt** [comment]
ci:forward

**Comment by nhpatt** [comment]
Resent because sF error here: https://github.com/liferay-headless/liferay-portal/pull/313

**Pull request closed without merging**